aten/src/ATen/SparseCsrTensorUtils.h

```diff
@@ -366,5 +367,36 @@ inline bool only_sparse_compressed_add_trivial_cases(
       });
 }
 
+inline Tensor to_type(Tensor input, ScalarType dtype) {
```

Comment:
Did you check whether we already have support for this conversion? We have the [`torch.Tensor.to`](https://pytorch.org/docs/master/generated/torch.Tensor.to.html?highlight=#torch.Tensor.to) operator.

Reply:
Please refer to the comment https://github.com/pytorch/pytorch/pull/99292#discussion_r1232201104.